addons/website/static/src/snippets/s_rating/options.js

```diff
@@ -126,17 +127,21 @@ options.registry.Rating = options.Class.extend({
      */
     _renderIcons: function () {
         const icons = {
-            'fa-star': 'fa-star-o',
-            'fa-thumbs-up': 'fa-thumbs-o-up',
-            'fa-circle': 'fa-circle-o',
-            'fa-square': 'fa-square-o',
-            'fa-heart': 'fa-heart-o'
+            'fa-star': 'fa-star',
+            'fa-thumbs-up': 'fa-thumbs-up',
+            'fa-circle': 'fa-circle',
+            'fa-square': 'fa-square',
+            'fa-heart': 'fa-heart'
         };
         const faClassActiveIcons = (this.iconType === "custom") ? this.faClassActiveCustomIcons : 'fa ' + this.iconType;
         const faClassInactiveIcons = (this.iconType === "custom") ? this.faClassInactiveCustomIcons : 'fa ' + icons[this.iconType];
         const $activeIcons = this.$target.find('.s_rating_active_icons > i');
         const $inactiveIcons = this.$target.find('.s_rating_inactive_icons > i');
         $activeIcons.removeClass().addClass(faClassActiveIcons);
         $inactiveIcons.removeClass().addClass(faClassInactiveIcons);
+        this.$target[0].setAttribute("aria-label", _t("%(amount)s out of %(total)s stars", {
+            amount: this.nbActiveIcons,
```

Comment:
If there is no rating (no yellow stars) , this code will output `"NaN"`  ;)

Suggestion:
```suggestion
            amount: this.nbActiveIcons || 0, 
```

Reply:
Thanks @stefanorigano 😅